Hardware/Hardware/CardReader/CardBrand.swift

```diff
@@ -82,3 +82,40 @@ private extension CardBrand {
         )
     }
 }
+
+extension CardBrand {
+    // The initializer for Bundle only works with class types.
+    // We use this class as a shortcut to find the bundle for the CardBrand enum type
+    private class _Bundle {}
+
+    private var iconURL: URL! {
+        Bundle(for: CardBrand._Bundle.self)
+            .url(forResource: iconName, withExtension: "svg")
+    }
+
+    /// Icon that represents the brand in SVG format
+    /// Since we mostly use these in HTML for receipts, we are reading the files directly instead of using UIImage
+    var iconData: Data! {
```

Comment:
Generally, I'm not a fan of force-unwrapping things, but I added a test to ensure every possible case has non-empty data. This should never be nil, event we can't enforce it at compile-time.